### ISSUE TITLE
workload detail missing envoy tab when Istio working with native sidecars

### ISSUE THREAD

**Issue by jshaughn** [kiali/kiali#7940]
In this case the sidecar proxy will be located in istioNativeSidecars.

**Comment by jshaughn** [comment]
[sergeyshevch](https://github.com/sergeyshevch), Thank you for the PR!  In general, every PR first needs an issue that describes the problem or request.  I created it for you, but for future contributions please start with an issue, so that you may receive initial feedback from the maintainers.

I think that Kiali may lack native sidecar support in a few places.  We also have this issue reported: https://github.com/kiali/kiali/issues/7909.  Do you have any interest in looking at that problem?

**Comment by jmazzitelli** [comment]
I just want to state the obvious here :) -- we should make sure whatever we do here, that we keep in mind the ambient work we've done where we do not expect sidecars to exist (i.e. make sure ambient support doesn't break)

**Comment by jshaughn** [comment]
> we keep in mind the ambient work

Good point.  I don't think this would have any negative effect wrt ambient.  This is about better detection of a sidecar proxy. It should not affect ambient, which would continue to not have a sidecar proxy.

**Comment by sergeyshevch** [comment]
I guess it shouldn't affect ambient in any way. Current problem that frontend not detect istio-sidecar if it located in initContainer, that lead to missing envoy tab in workload details view
